nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/test/collections/GetCollectionsTest.java

```diff
@@ -82,7 +86,8 @@ public void testGetCollections() throws Exception {
 
         // Search several collections
         chain = new OperationChain("test-chain-2");
-        chain.add(GetCollectionsOperation.ID).set("searchTerm", COLLECTION_NAME + "%");
+        chain.add(GetCollectionsOperation.ID)
+             .set("searchTerm", COLLECTION_NAME.subSequence(0, COLLECTION_NAME.length() - 3));
```

Comment:
Is there a point using `subSequence` instead of `substring` ?

Reply:
indeed substring ensures continuity...i'll go for it since we want to select a prefix